src/contracts/core/DelegationManager.sol

```diff
@@ -264,6 +264,34 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         });
     }
 
+    function queueWithdrawals(
+         IStrategy[][] calldata strategies,
```

Comment:
i think we should create a `QueuedWithdrawaParam` struct and the param here should be `QueuedWithdrawaParam[]`